packaging/bin/engine-backup.sh.in

```diff
@@ -173,7 +173,8 @@ BACKUP_PATHS="/etc/ovirt-engine
 # Add /var/lib/ovirt-engine except a few
 VAR_LIB_OVIRT_ENGINE_EXCLUSIONS="/var/lib/ovirt-engine/backups
 /var/lib/ovirt-engine/jboss_runtime
-/var/lib/ovirt-engine/ansible-runner-service.log"
+/var/lib/ovirt-engine/ansible-runner-service.log
```

Comment:
We no longer need ansible-runner-service.log, right?

Reply:
It's only a filter list. No harm in filtering it out even if it does not exist.
And it might exist from previous versions.

Reply:
OK